### PR TITLE
Added SGI-Content features

### PR DESCRIPTION
**Pull request by lsemp3d**
Signed-off-by: lsemp3d <58790905+lsemp3d@users.noreply.github.com>

### CODE REVIEW

**Review by mcphedar — CHANGES_REQUESTED** [review]
Hey Luis. We don't want to remove the terrain and vegetation systems. Can you add those back in ?

File: features/sigjson/sig-content-features.json [hunk 1]
```diff
@@ -3,27 +3,335 @@
...
+         "Functional": "",
+         "Content": "",
+         "Code/API": "",
+         "Performance": "",
+         "Platform": "",
+         "Link": "",
+         "Doc": "",
+         "Notes": ""
+ 
+      }
```
Comment (hultonha) [inline features/sigjson/sig-content-features.json:146]: I've listed White Box as it's own subsystem with various features so we might want to remove this from here?

File: features/sigjson/sig-content-features.json [hunk 2]
```diff
@@ -3,27 +3,335 @@
...
-            "Content": "1",
-            "Code/API": "3",
-            "Performance": "3",
-            "Platform": "2;",
+            "Feature": "",
+            "Functional": "",
+            "Content": "",
+            "Code/API": "",
+            "Performance": "",
+            "Platform": "",
```
Comment (hultonha) [inline features/sigjson/sig-content-features.json:322]: It looks like the original values for this have been wiped, I'd suggested keeping the original changes

File: features/sigjson/sig-content-features.json [hunk 3]
```diff
@@ -3,27 +3,335 @@
...
-            "Content": "5",
-            "Code/API": "5",
-            "Performance": "3",
-            "Platform": "1;",
+            "Feature": "",
+            "Functional": "",
+            "Content": "",
+            "Code/API": "",
+            "Performance": "",
+            "Platform": "",
```
Comment (hultonha) [inline features/sigjson/sig-content-features.json:334]: Same as above

**Review by monroegm — APPROVED** [review]

**Pull request merged**